doc/user_guide/eds.rst

```diff
@@ -525,31 +533,49 @@ Quantification can be applied from the intensities (background subtracted) with
     >>> kfactors = [1.450226, 5.075602] #For Fe Ka and Pt La
     >>> bw = s.estimate_background_windows(line_width=[5.0, 2.0])
     >>> intensities = s.get_lines_intensity(background_windows=bw)
-    >>> weight_percent = s.quantification(intensities, kfactors, plot_result=True)
+    >>> atomic_percent = s.quantification(intensities, method='CL', kfactors)
     Fe (Fe_Ka): Composition = 4.96 weight percent
     Pt (Pt_La): Composition = 95.04 weight percent
 
-The obtained composition is in weight percent. It can be changed transformed into atomic percent either with the option :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.quantification`:
+The obtained composition is in atomic percent, by default. However, it can be transformed into weight percent either with the option :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.quantification`:
 
 .. code-block:: python
 
     >>> # With s, intensities and kfactors from before
-    >>> s.quantification(intensities, kfactors, plot_result=True,
-    >>>                  composition_units='atomic')
+    >>> s.quantification(intensities, method='CL',kfactors,
+    >>>                  composition_units='weight')
     Fe (Fe_Ka): Composition = 15.41 atomic percent
     Pt (Pt_La): Composition = 84.59 atomic percent
 
-either with :py:func:`~.misc.material.weight_to_atomic`. The reverse method is :py:func:`~.misc.material.atomic_to_weight`.
+or with :py:func:`~.misc.material.atomic_to_weight`. The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
 
 .. code-block:: python
 
-    >>> # With weight_percent from before
-    >>> atomic_percent = hs.material.weight_to_atomic(weight_percent)
+    >>> # With atomic_percent from before
+    >>> weight_percent = hs.material.atomic_to_weight(atomic_percent)
+
+The zeta-factor method needs both the beam_current (in nA) and the dwell time (referred to as real_time in seconds) in order to provide an accurate quantification. Both of the these parameters can be set to in the metadata of the spectrum image using:
+
+..code-block:: python
+
+    >>> s.metadata.Acquisition_instrument.TEM.beam_current = 0.5
+    >>> s.metadata.Acquisition_instrument.TEM.Detector.EDS.real_time = 1.5
+
+If these are not set the code will produce an error stating which parameter has been forgotten.
+
+The cross section method needs the beam_current, real_time and pixel width in order to provide an accurate quantification, which can be set using:
+
+..code-block: python
+
+    >>>> s.axes_manager[0].scale = 0.025
+    >>>> s.axes_manager[1].scale = 0.025
+
+If the pixel width is not set, the code will still run with the default value of 1nm with a warning message to remind the user that this is the case.
 
```

Comment:
@k8macarthur I think you should do what @siriagus did in his local branch and add these parameters to the already existing set_microscope_parameters() function.

Then users will be able to do for example set_microscope_parameters(pixel_width=0.025) so the documentation you've written here will need to reflect that - this will be much cleaner all round, I think you're safe to just go for it.